### PR TITLE
add pvc

### PR DESCRIPTION
**Pull request by fabiangotzen**
The Lamda-Csv-Importer needs access to the lamda data, directories can be found here: 

https://alv-ch.atlassian.net/browse/DF-1104

Used a similar approach as for the kofax-exporter, please doublecheck

### CODE REVIEW

**Comment by seco-mgabor** [comment]
Fabian, as a normal user, you cannot create a PV. Only the cluster admin can do that and he removed our admin rights. Please contact Roland if you need assistance in this topic.
He also has to create manually a mapping in the NFS in order for us to use it as PV in OpenShift.
It's only NFS storage class that needs this special "treatment".

File: openshift/alv-lamda-csv-importer/deployment-config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+kind: List
```
Comment (birom) [inline openshift/alv-lamda-csv-importer/deployment-config.yml]: Deployment config is not needed. The SCDF k8s deployer starts the new pod.

Comment (fabiangotzen) [reply openshift/alv-lamda-csv-importer/deployment-config.yml]: removed

File: openshift/alv-lamda-csv-importer/pvc-csv-base-resource.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: csv-base-resource
+  labels:
+    app: alv-lamda-csv-importer
+spec:
+  accessModes:
+    - ReadOnlyMany
+  resources:
+    requests:
+      storage: 256Mi
+  storageClassName: "nfsstorage"
```
Comment (cinhtau) [inline openshift/alv-lamda-csv-importer/pvc-csv-base-resource.yml]: The claim need the volume name to bind it.

```
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: csv-base-resource
  labels:
    app: alv-lamda-csv-importer
spec:
  accessModes:
    - ReadOnlyMany
  resources:
    requests:
      storage: 256Mi
  storageClassName: "nfsstorage"
  volumeName: "pvnfs-lamda-dev"
```

**Pull request merged**